### PR TITLE
Two-fer rewrite - draft

### PR DESCRIPTION
**Pull request by kytrinyx**
Here's an attempt at adding more context around the term two-fer, and adding a touch of story.

### CODE REVIEW

**Review by ErikSchierboom — APPROVED** [review]
This is lovely!

File: exercises/two-fer/introduction.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+In some dialects of English, when you say "two for" quickly or in casual conversation, it sounds like "two fer".
+
+Imagine a store that sells a particular t-shirt for $10.
+For Valentine's day, they might provide a special offer for couples where if you buy two shirts of the same design (even if the size is different), then you get the second one for free.
+This would be a two-fer, since you get two for the price of one.
+
+Similarly, some concert venues will offer a discount if you buy two tickets instead of just one.
+
+In this exercise you have bought two cookies for the price of one.
+You are going to (very generously) share with a friend.
```
Comment (iHiD) [inline exercises/two-fer/introduction.md:16]: I think we can remove the other examples, and just stick with the cookie example.

```suggestion
Imagine a food store that has a Valentine's discount where you can buy two cookies for the price of one ("two-fer one!"). You've gone for the offer and (very generously) decided to share the extra cookie with a friend!
```

Comment (iHiD) [reply exercises/two-fer/introduction.md:16]: (see my first point on https://github.com/kytrinyx/problem-specifications/pull/2#issuecomment-1433452680 for context)

**Comment by iHiD** [comment]
I really like the explanation. As per my gigasecond comments I think it should be slightly shorter though.

File: exercises/two-fer/introduction.md (hunk 1 above)
Comment (kytrinyx) [reply exercises/two-fer/introduction.md]: I like that. I erred on the side of more examples and context than less, but I really like the shorter version that just goes straight to the cookies.

**Comment by iHiD** [comment]
>  Do you think the instructions should be shorter as well? 

No, I think that adds clarity, which is basically always good in my mind.

I think the aim for the introduction should be fun and context without noise. And for the instructions, the aim should be clarity.

**Comment by kytrinyx** [comment]
Cool, that's useful.

**Comment by kytrinyx** [comment]
I've shorted the intro a bit more. @iHiD would you take another look?

File: exercises/two-fer/introduction.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Introduction
+
+In some dialects of English, when you say "two for" quickly or in casual conversation, it sounds like "two fer".
+Two-for-one is a way of saying that if you buy one, you also get one for free.
```
Comment (iHiD) [inline exercises/two-fer/introduction.md:4]:
```suggestion
In some English accents, when you say "two for" quickly, it sounds like "two fer".
Two-for-one is a way of saying that if you buy one, you also get one for free. 
So the phrase "two-fer" often implies a two-for-one offer.
```

- Change dialect to accent.
- Removed "casual conversation"
- Added a final sentence to bring the paragraph together.

**Comment by kytrinyx** [comment]
Opened PR upstream.

**Pull request closed without merging**